.github/workflows/unit-tests.yml

```diff
@@ -43,18 +59,17 @@ jobs:
 
   coverage:
     name: Code coverage
-    needs:
-      - test
```

Comment:
Why we remove this one? Could we revert?

Reply:
I thought we need `needs: check_docs_only` in the file and since `needs: - test` was not used, I thought we could remove it.